.travis.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -eo pipefail
+
+run() {
+  run_main_task
+  report_coverage
+  documentation
+  check_requiring_libraries
+}
+
+# Check requiring libraries successfully.
+# See https://github.com/rubocop-hq/rubocop/pull/4523#issuecomment-309136113
+check_requiring_libraries() {
+  logged ruby -I lib -r rubocop -e 'exit 0'
+}
+
+# Running YARD under jruby crashes so skip checking the manual.
+documentation() {
+  if ! is_jruby; then
+    logged bundle exec rake documentation_syntax_check generate_cops_documentation
+  fi
+}
+
+is_jruby() {
+  [ "$(ruby -e "puts RUBY_ENGINE == 'jruby'")" == 'true' ]
```

Comment:
Note that I'm actually spawing a few more ruby processes here, since we need that to be able to check the `RUBY_ENGINE` constant from bash. But I think this is fine: these are small first level ruby subprocesses. In https://github.com/rubocop-hq/rubocop/compare/master...deivid-rodriguez:debug_ci_fork I avoid these processes by making sure the `JRUBY_OPTS` environment variable is set only on `jruby` matrix entries, but I think that can be done only if necessary on follow up PRs.